pytest_factoryboy/fixture.py

```diff
@@ -48,13 +49,17 @@ def make_fixture(name, module, func, args=None, related=None, **kwargs):
     return fixture
 
 
-def register(factory_class, _name=None, **kwargs):
+def register(factory_class=None, _name=None, **kwargs):
```

Comment:
I am thinking if it's a good idea to allow passing `_name` as a positional argument, maybe we should force the developer to always pass it as a keyword argument:

```python
@register(_name='my_instance')
class MyFactory(...):
    ...
    
    
register(MyFactory, _name='my_other_instance')
```

Please let me know what you think about it because this would be a breaking change.

Reply:
I think that `_name` should be allowed to be pass as a positional argument, as it's quite clear what it does without being to explicit:
`register(MyFactory, "my_other_instance")`

I think we should instead eventually pass the kwargs in a container dict, rather than as **kwargs:
```python
register(FooFactory, model_kwargs={"bar": LazyFixture("second_bar")})
```
so that the `_name` param can actually become `name` without underscore, and there will be no option of clashing with the model kwargs.

But this would require a backwards incompatible change I think. Unless we try to do a very hacky signature (which I'm not a fan of)

Reply:
That's a great idea, but because it's a breaking change I think we should implement it in separate PR.